Create MediaMail calculator

kjriga opened the pull request:
Porting over the MediaMail calculator from TJ. I added a seed file to ensure the required shipping category can be made if desired, to use this calculator.

benjaminwil commented on lib/solidus_usps/calculator/media_mail.rb:
```diff
@@ -0,0 +1,31 @@
+module SolidusUsps
+  module Calculator
+    class MediaMail < SolidusUsps::Calculator::Base
+      MEDIA_MAIL_CATEGORY_NAME = "Media Mail".freeze
```
Nit: This constant is defined the `MediaMail` class so `MEDIA_MAIL_` is a redundant prefix in the end. Suggestion:

```diff
- SolidusUsps::Calculator::MediaMail::MEDIA_MAIL_CATEGORY_NAME
+ SolidusUsps::Calculator::MediaMail::CATEGORY_NAME
```

benjaminwil reviewed (approved):
Looks good. I have a couple suggestions:

- Less redundant constant name (see additional comment)
- Fix the missing commit message (`wip`)

kjriga commented:
Fixed Benjamin's comments. I also updated this calculator to preemptively add the new `mail_class` method so it doesn't get missed, since that change is in [PR](https://github.com/SuperGoodSoft/solidus_usps/pull/12) at the same time as this.
